sim/common/itemhelpers/weaponprocs.go

```diff
@@ -8,7 +8,7 @@ import (
 
 // Create a simple weapon proc that deals damage.
 func CreateWeaponProcDamage(itemId int32, itemName string, ppm float64, spellId int32, school core.SpellSchool,
-	dmgMin float64, dmgRange float64, bonusCoef float64, defType core.DefenseType) {
+	dmgMin float64, dmgRange float64, bonusCoef float64, defType core.DefenseType, spellFlagExlcude core.SpellFlag) {
```

Comment:
Looks like there's a typo in the param name here

Reply:
fixed